ui/app/src/utils/constants.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2007-2021 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const NEW_MASK = 0b0000000000000000000000000000000000000000000000000000000000000001;
+const MODIFIED_MASK = 0b0000000000000000000000000000000000000000000000000000000000000010;
+const DELETED_MASK = 0b0000000000000000000000000000000000000000000000000000000000000100;
+const LOCKED_MASK = 0b0000000000000000000000000000000000000000000000000000000000001000;
+const SYSTEM_PROCESSING_MASK = 0b0000000000000000000000000000000000000000000000000000000000010000;
+const IN_WORKFLOW_MASK = 0b0000000000000000000000000000000000000000000000000000000000100000;
+const SCHEDULED_MASK = 0b0000000000000000000000000000000000000000000000000000000001000000;
+const STAGED_MASK = 0b0000000000000000000000000000000000000000000000000000000010000000;
+const LIVE_MASK = 0b0000000000000000000000000000000000000000000000000000000100000000;
+const TRANSLATION_UP_TO_DATE_MASK = 0b0000000000000000000000000000000000000000000000000000001000000000;
+const TRANSLATION_PENDING_MASK = 0b0000000000000000000000000000000000000000000000000000010000000000;
+const TRANSLATION_IN_PROGRESS_MASK = 0b0000000000000000000000000000000000000000000000000000100000000000;
+const SUBMITTED_MASK = 0b0000000000000000000000000000000000000000000000000001000000000000;
```

Comment:
- Name all as STATE_MASK_{state name} (e.g. STATE_MASK_NEW, etc)
- export them all
- I don't think we need all the leading zeros after the `0b`
- have a look around other utils to see if there are constants that should be moved here

Reply:
not sure about this `I don't think we need all the leading zeros after the 0b`